src/pug/blog/report_25.pug

```diff
@@ -0,0 +1,98 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #25
+
+block content
+  h2 Progress report #25
+  +news_header('13 September 2024', 'report_24', '')
+
+  h3 Wireless RF prototypes 6 and 7
+  p We continued optimizing antenna performance by doing #[em "trim tuning"], adding a long copper stub on the antenna's end, and then taking measurements while progressive scratching off the copper and reducing the stub length.
+
+  p In the #[em v6] iteration the antenna was placed horizontally, but unfortunately the antenna was detuned so much that the center frequency was under the target no matter the stub length.
```

Comment:
Is all of this really relevant? Why did we not follow the reference design in the first place ...
I would shorten all of this significantly.